.travis.yml

```diff
@@ -4,6 +4,10 @@ node_js: "5.5.0"
 
 sudo: false
 
+env:
+  GITHUB_REPO: github.com/18F/fec-style
```

Comment:
Agreed--seems like changing the script would make sense. Something like:

```
git push --force --quiet "https://x-api-token:${GITHUB_TOKEN}@github.com/${TRAVIS_REPO_SLUG}.git" ...
```

NBD, obv--just nit-picking.